kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -107,7 +107,7 @@ class BatchJobSubmission(session: KyuubiBatchSessionImpl, batchRequest: BatchReq
         Thread.sleep(applicationCheckInterval)
       }
       val state = applicationStatus.get(ApplicationOperation.APP_STATE_KEY)
-      if (state == "KILLED" || state == "FINISHED") {
+      if (state == "KILLED" || state == "FINISHED" || state == "FAILED") {
         process.destroyForcibly()
         throw new RuntimeException("Batch job failed:" + applicationStatus.get.mkString(","))
```

Comment:
why we throw RuntimeException if the application state is `FINISHED`?

Reply:
my mistake